modules/command/dc/dc-challenges-command.js

```diff
@@ -43,10 +43,6 @@ class DCChallengesCommand extends Command {
                     return;
                 }
 
-                challenge.status = 'IN_PROGRESS';
-                await challenge.save({ fields: ['status'] });
-
-
```

Comment:
What are we doing about challenge status? It doesn't seem to be updated in all cases, should we delete it?

Reply:
We are using challenge status in project and we are updating it where we need, but this line shouldn't be deleted it's mistake.